Disabling Static Files Authorization for external software (#3064)
opencast/opencast: docs/guides/admin/docs/configuration/serving-static-files.md

An alternative method to this is Opencast's [token based authorization system](stream-security.md) which allows (and
requires) to defer all security checks to external systems.

> With Opencast 10, a new default configuration for static files authorization was introduced.

Reviewer: Avoid mentioning specific Opencast versions. The docs are specific to the versions anyway:

```suggestion
Static files authorization is active by default.
```

> For some external software, like Ilias, it is required that all of your media is accessable to every user. If that is not the case, the static files authorization must be disabled by setting the `authentication.required` option in `etc/org.opencastproject.fsresources.StaticResourceServlet.cfg` to `false`.  

Reviewer: ```suggestion
For some external software, like the ILIAS plugin, it is required to set `authentication.required` to `false`.  
```

The paragraph in line 29 talks about where to find the “configuration options for requiring authentication”. Maybe move this paragraph directly after that one? Then, we also do not need to mention the configuration file again.
